### ISSUE TITLE
Feature-Request: v-model format

### ISSUE THREAD

**Issue by nemeth-it** [mariomka/vue-datetime#173]
All I found here about **format** seems related how the `vue-datetime`-component should render it. But what I still missing here is a possibility to define input and output-format for the **v-model**, too.

I will now workaround by converting the date in model. But it would be nice feature if you can define the format for the **v-model** in the future, too.

**Comment by mariomka** [comment]
Yep, right now the only way is to convert after the date is selected.

**Comment by lahmacun** [comment]
Hello @mariomka , first of all thanks for this project. It's good.
Is there any development about this feature?
